src/Logger.php

```diff
@@ -110,7 +115,7 @@ public function __construct(array $targets = [])
      *
      * @return string The text display of the level.
      */
-    public static function getLevelName($level): string
+    public static function level($level): string
```

Comment:
It doesn't transform level in any way but validate it. Maybe worth renaming it to `validateLevel($level): void`?